alertlogic/services.py

```diff
@@ -0,0 +1,193 @@
+from service import Service
+
+
+class Services:
```

Comment:
looked deeper, i don't think we need this class anymore. Context holds credentials and region, we can just instantiate our classes whenever required:
```python
from alertlogic.services import Otis
otis = Otis(context)
...
 response = otis.write_an_option(
                account_id=args["account_id"],
                json=new_option
            )
```

Reply:
actually! Session holds region as well, let's get rid of the context thing, let just Session be.